modin/backends/pandas/query_compiler.py

```diff
@@ -2573,6 +2555,39 @@ def _groupby_dict_reduce(
         drop=False,
         **kwargs,
     ):
+        """
+        Group underlying data and apply aggregation functions to each group of the specified column/row.
+
+        This method is responsible to perform dictionary groupby aggregation for such functions,
+        that can be implemented via MapReduce approach.
+
+        Parameters
+        ----------
+        by : PandasQueryCompiler, column or index label, Grouper or list of such
+            Object that determine groups.
+        axis : {0, 1}
+            Axis to group and apply aggregation function along.
+            0 is for index, when 1 is for columns.
+        agg_func : dict(label) -> str
+            Dictionary that maps row/column labels to the function names.
+            **Note:** specified functions have to be supported by ``modin.data_management.functions.GroupbyReduceFunction``.
```

Comment:
what do you mean by "supported" here?

Reply:
This means that the passed function has to be splittable into a map and reduce functions. There is a list of known splittable functions in [`GroupbyReduceFunction.groupby_reduce_function`](https://github.com/modin-project/modin/blob/6bc748e6e835ecfb6e9779aa93cbd47c2f412511/modin/data_management/functions/groupby_function.py#L269) that is used to split passed functions. Roughly saying, "supported" means that the function is in that list.

Added this info to the docstring.